src/scales.jl

```diff
@@ -42,6 +46,19 @@ Base.IteratorEltype(::Type{Scale{T}}) where {T} = Base.HasEltype()
 Base.eltype(::Type{Scale{T}}) where {T} = T
 
 
+function Base.getindex(s::Scale{Pitch}, n::Int)
+
+    octave = n ÷ length(s.notes)
+    n = n % length(s.notes)
+    if n < 0
+        n += length(s.notes)
+        octave -= 1
+    end
+
+    return Pitch(PitchClass(s.notes[n + 1]), s.notes[n + 1].octave + octave)
+end
```

Comment:
Why zero-based indexing?

Reply:
What else would make sense? That way positive and negative are symmetric around the tonic.

Reply:
1-based indexing is not only more common in Julia, but it also seems more common. If you poll 100 musicians with “what is the 5th note in the C major scale”, how many do you think would answer “A”?

Reply:
See e.g. https://en.m.wikipedia.org/wiki/Degree_(music)

Reply:
Nor do understand the point of positive and negative indices being symmetric around the tonic. The intervals aren’t symmetric, nor are the musical functions of the notes.

Reply:
Good points, thanks! Let me rethink